### DISCUSSION TITLE
Pulling release caused my HASS to crash

### DISCUSSION THREAD

**Discussion by MrKuenning** [hassio-addons/addon-zwavejs2mqtt#122] [General]
Just an FYI. After months of using Z-Wave JS to MQTT with zero issues. A few days ago my HASSIO VM started crashing. It would last anywhere from 1 minute to 25 minutes before suddenly powering off. I looked everywhere for a cause, from VCenter to HASS Logs but there was no culprit.

Suddenly I noticed that both Z-Wave JS and Z-Wave JS to MQTT were installed and running. Every time I would stop Z-Wave JS and even uninstall it. After the VM powered back on it was back. Then I also noticed that Z-Wave JS to MQTT had a generic puzzle icon instead of the normal Z-Wave icon. Finally out of desperation I uninstalled Z-Wave JS to MQTT. Only to find it was no longer listed in the Supervisor add-on store. Instantly my system stopped powering itself off. I tried in vain to figure out where Z-Wave JS to MQTT went, but it was completely gone from the store. So I reconfigured my Z-Wave to use the built-in Z-Wave JS. Today I checked and it's listed in the store again, and I noticed in the changelog that it says "The previous release was withdrawn" I don't know if that means you withdrew it or the HASS team did, but it seems like somehow pulling the release broke my container enough to force HASS to install the default Z-Wave JS, and having both enabled caused instability and crash.

I am not complaining or pointing fingers, and I greatly appreciate the products you and others create for the community. But I wanted you to know about my experience. Especially if it happened to others too. And if it was caused by pulling a release, maybe in the future you could just push a replacement instead.

Brady

**Comment by frenck** [comment]
I withdrew it, this is the community add-ons repository which is maintained by me, not the Home Assistant Project.

Pulling an add-on does not affect existing installations in general (I'm a user on my own production instance myself as well).


See also here: <https://twitter.com/hassioaddons/status/1379418322903691265>
